docs/components/concepts/process-instance-creation.md

```diff
@@ -27,28 +27,23 @@ This command creates a new process instance and immediately responds with the pr
 ![create-process](assets/create-process.png)
 
 <details>
-  <summary>Code example</summary>
-  <p>
-Create a process instance:
+   <summary>Create a process instance via Camunda 8 REST API</summary>
+   <p>
 
 ```
-zbctl create instance "order-process"
+curl -L 'http://localhost:8080/v2/process-instances' \
+-H 'Content-Type: application/json' \
+-H 'Accept: application/json' \
+-d '{
+  "processDefinitionKey": "2251799813685249”,
+  "processDefinitionVersion": 1
+}'
```

Comment:
❓ Shouldn't this (like a couple of others here) be a POST request? So we're missing `-X POST`, I guess? Maybe this is done by some cURL magic I don't know about here though, happy to learn something new 🤷

Reply:
Yesssss!

Reply:
I would love to link you to a man page, but apparently, that's not how those docs work 💩 

https://stackoverflow.com/questions/18820100/how-does-data-option-work-in-curl